src/command/helpModule.ts

```diff
@@ -40,6 +41,34 @@ export default class HelpModule extends Module {
             );
         }, initialEmbed);
 
-        await msg.channel.send(embed);
+        const fallbackContent = modules.reduce((message, module) => {
+            const commands = _commands.filter(
+                command => command.module === module
+            );
+            return message.concat(
+                module.constructor.name,
+                "\n",
+                commands
+                    .map(command => {
+                        const name = command.id.split("/")[1];
+                        const description = command.description
+                            ? `: ${command.description}`
+                            : "";
+                        return `${prefix}${name}${description}`;
+                    })
+                    .join("\n") + "\n"
+            );
+        }, new Array());
```

Comment:
`[]`?

Reply:
I've tried adding this, but the typescript compiler would throw errors which is how I ended up at that solution.

Reply:
Messed around with it a bit and it looks like `[] as string[]` is the winner.

Reply:
got it